keras_rcnn/models.py

```diff
@@ -33,12 +33,15 @@ def __init__(self, inputs, features, heads, rois):
         proposals                        = keras_rcnn.layers.ObjectProposal(maximum_proposals=rois)([im_info, rpn_regression, rpn_classification])
         rpn_labels, rpn_bbox_reg_targets = keras_rcnn.layers.ProposalTarget()([gt_boxes, im_info])
 
+        rpn_classification_loss = keras_rcnn.layers.ClassificationLoss(anchors=num_anchors, name="rpn_classification_loss")([rpn_labels, rpn_classification])
+        #rpn_regression_loss     = keras_rcnn.layers.RegressionLoss(name="rpn_bbox_loss")([rpn_bbox_reg_targets, rpn_regression])
```

Comment:
I did, yeah. It was giving me errors because of wrong shapes etc. Figured lets discuss it first, then apply the same logic to all losses :)